app/views/Project/EntryEdit/index.tsx

```diff
@@ -1423,26 +1424,42 @@ function EntryEdit(props: Props) {
                                     )}
                                     contentClassName={styles.reviewContent}
                                 >
-                                    <VirtualizedListView
-                                        itemHeight={360}
-                                        keySelector={entryKeySelector}
-                                        renderer={EntryInput}
-                                        data={formValue.entries}
-                                        direction="vertical"
-                                        spacing="comfortable"
-                                        rendererParams={entryDataRendererParams}
-                                        filtered={false}
-                                        errored={false}
-                                        pending={false}
-                                        emptyIcon={(
-                                            <Kraken
-                                                variant="search"
+                                    {(formValue?.entries?.length ?? 0) > 0
```

Comment:
Doesn't virtualize list view support empty component?

Reply:
Message component was implemented as it covers the sufficient container area ans also with the reason that "emptyIcon" component of VirtualizedListView was not supporting the Kraken properly.